src/SerializeSourcemap.cc

```diff
@@ -182,6 +208,26 @@ sos::Object WrapPayloadSourcemap(const SourceMap<Payload>& payload)
     // Schema
     payloadObject.set(SerializeKey::Schema, WrapSourcemap(payload.schema));
 
+    // Content
+    sos::Array content;
+
+    /// Attributes
```

Comment:
It signalizes a deeper level. `content` is first level, and `attributes` is inside `content`.